src/test/java/chess/domain/piece/KnightTest.java

```diff
@@ -0,0 +1,41 @@
+package chess.domain.piece;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import chess.domain.CachedPosition;
+import chess.domain.board.Position;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Knight 테스트")
```

Comment:
```suggestion
@DisplayName("Knight는")
```
처럼 쓰는게 조금 더 문장 흐름에 자연스러울 것 같네요 :)

Reply:
수정하겠습니다.